modules/ossm-configuring-the-threescale-wasm-auth-module.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-configuring-the-threescale-wasm-auth-module_{context}"]
+= Configuring the `threescale-wasm-auth` module
+
+As the OpenShift administrator, configure the `threescale-wasm-auth` module to authorize HTTP requests to 3scale API Management through an application binary interface (ABI). The ABI defines the interaction between host and the module, exposing the hosts services, and allows you to use the module to process proxy requests.
+
```

Comment:
```suggestion
Cluster administrators on {product-title} can configure the `threescale-wasm-auth` module to authorize HTTP requests to 3scale API Management through an application binary interface (ABI). The ABI defines the interaction between host and the module, exposing the hosts services, and allows you to use the module to process proxy requests.

```
suggestion, sorry I missed this before; where possible use the product name / version attributes https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version